docs/application/native/guides/multimedia/media-playback.md

```diff
@@ -38,21 +38,17 @@ You can [play audio](#play_audio) and control functions for using audio resource
 
 The supported audio formats include MP3, AAC, WMA, M4A, 3GA, WAV, and AMR. The available formats depend on the target device.
 
-You can use multiple instances of the player to play several audio data streams concurrently. This means that your application can [play multiple uncompressed audio files](#play) at the same time.
+S/W and H/W audio codec could be selected by player API and the default codec type should be checked by calling player_get_audio_codec_type().
```

Comment:
Suggest rephrasing, please verify:

```suggestion
Software and hardware code can be selected using the player API. You must verify the codec by calling `player_get_audio_codec_type()`.
```

Reply:
Hello, @surbhig12 
The "S/W and H/W audio codec" is changed to "Software and hardware code", and I hope you to check the update again. With this update, isn't the "audio codec" become "code"?

Reply:
If the 'codec' is not normal, we can replace it to 'decoder'.

Reply:
@eunhaechoi,

In my opinion, in this scenario decoder may not be right. 
codec means: A device or program that compresses data to enable faster transmission and decompresses received data. 
decoder means: a device capable of converting audio or video signals into a different form, for example from digital to analogue.
In this scenario, I think we are referring to codec

Please correct me if I am wrong

```suggestion
Software and hardware audio codec can be selected using the player API. You must verify the codec by calling `player_get_audio_codec_type()`.
```

Reply:
Additional question to @surbhig12 

**Original :** the default codec type should be checked by calling player_get_audio_codec_type().
**Suggested :** You must verify the codec by calling `player_get_audio_codec_type()`.

According to the API Reference, `player_get_audio_codec_type()` is called to get the default codec type information of the audio decoder. As the default codec type can be different depending on the device, I think the original sentence is focusing on "To check the information what the default codec type is" as the sentence itself.
However, the suggested sentence is focusing on more general meaning ("verify codec"), so to me, it seams the meaning get blurred a little bit. Would you please check again the sentence?

If I understood something wrong, please let me know.

**Reference :** http://10.113.164.110/native/5.5/mobile/group__CAPI__MEDIA__PLAYER__MODULE.html#ga5b88c6383ed311fdd6ef2e7530505d17
http://10.113.164.110/native/5.5/mobile/group__CAPI__MEDIA__PLAYER__MODULE.html#gacb9bebbd139f12bb45ad2a8ddd332da4

Reply:
@mijin85cho, thank you for the detailed explanation.
Would the following be okay:

Software and hardware audio codec can be selected using the player API. You must verify the codec type by calling `player_get_audio_codec_type()`.

Reply:
@surbhig12 
- Well... I think "verify" is too heavy to the meaning I understood because it is _"to get the default codec type information / It can also fail to get the default codec type because of wrong parameters or something else"_, not _"to confirm the codec type information."_
As the "verify" sounds like "confirming what kind of codec type information and whether it is valid/or not" to me, if possible I hope to use "check" rather than "verify".
Anyway if you insist, we can use "verify" also.

- I thought the "default codec type" was the point, so it would be better to add back the "default" to your suggestion.

Reply:
@mijin85cho, sorry for the miss and thank you for correcting me:

Please find the updated sentence:

Reply:
@surbhig12 Thanks for considering my opinion. I think the latest sentence is good.